Source/Core/Core/IOS/Network/Net.cpp

```diff
@@ -527,7 +527,17 @@ static int inet_pton(const char* src, unsigned char* dst)
 }
 
 // Maps SOCKOPT level from native to Wii
-static unsigned int opt_level_mapping[][2] = {{SOL_SOCKET, 0xFFFF}};
+int32_t mapWiiSockOptToNative(uint32_t level)
+{
+  switch (level)
+  {
+  case 0xFFFF:
+  	return SOL_SOCKET;
+    
+  default:
+    return -1;
+  }
+}
 
 // Maps SOCKOPT optname from native to Wii
 static unsigned int opt_name_mapping[][2] = {
```

Comment:
It would be nice if we could make this consistent with `mapWiiSockOptToNative` (or at least something nicer than `[][2]`).